compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/Fir2IrClassifierStorage.kt

```diff
@@ -141,6 +142,14 @@ class Fir2IrClassifierStorage(
         }
     }
 
+    internal fun getCachedIrClass(classId: ClassId): IrClass? {
```

Comment:
As I can see, this function is called *only* with local class id. So may be it's worth renaming it to `getCachedLocalClass` and removing the second if branch.

Reply:
Right; renamed and also added `require` to make sure.